orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -355,7 +405,38 @@ Status TrainingRunner::PrepareFeedNamesAndFeeds(const SessionMode mode,
     feeds.push_back(event_id);
   }
 
-  // Create feed of recorded event in forward pass.
+  // Create feed of the second waited event in forward pass.
+  if (!pipeline_context_.forward_waited_event_after_recv_name.empty()) {
+    ORT_ENFORCE(params_.pipeline_parallel_size > 1);
+    feed_names.push_back(pipeline_context_.forward_waited_event_after_recv_name);
+    // TODO: Generate this event ID from the scheduler.
```

Comment:
>// TODO: Generate this event ID from the scheduler. [](start = 4, length = 51)

remove?

Reply:
Sure.

---
In reply to: [425503975](https://github.com/microsoft/onnxruntime/pull/3823#discussion_r425503975) [](ancestors = 425503975)